c_common/front_end_common_lib/src/recording.c

```diff
@@ -633,6 +642,11 @@ bool recording_initialize(
     simulation_dma_transfer_done_callback_on(
             RECORDING_DMA_COMPLETE_TAG_ID, recording_dma_finished);
 
+    // update word tracker for anything else
+    *words_read += (
+        (sizeof(recording_data_t) / WORD_TO_BYTE_CONVERSION) +
+        (N_WORDS_USED_OUTSIDE_STRUCT_PER_REGION * n_recording_regions));
+
```

Comment:
This might be better as bytes read, then you don't need to do so much work here.  Another possibly simpler option is to pass in a ```void **``` or a ```uint8_t **``` as the address to read from, then you can just update the pointer to the region.

-

Reply:
reason for words instead of bytes, is that the guy using it is using it as an array offset. someone needs to do the divide. up to you whom. 

I guess you could do a return (param out) for the address and then the neuron recorder would be able to use it without the offset..... youd need to change everywhere who reads recording_init to change the param type again..... but doable.